src/java/org/apache/cassandra/repair/autorepair/AutoRepairConfig.java

```diff
@@ -328,6 +347,43 @@ public void setRepairSessionTimeout(RepairType repairType, String repairSessionT
         getOptions(repairType).repair_session_timeout = new DurationSpec.IntSecondsBound(repairSessionTimeout);
     }
 
+    @VisibleForTesting
+    static IAutoRepairTokenRangeSplitter newAutoRepairTokenRangeSplitter(AutoRepairConfig.RepairType repairType, ParameterizedClass parameterizedClass) throws ConfigurationException
+    {
+        try
+        {
+            Class<?> tokenRangeSplitterClass;
```

Comment:
nit: to avoid casting below
```suggestion
            Class<? extends IAutoRepairTokenRangeSplitter> tokenRangeSplitterClass;
```